src/components/Sections/Events/Filters.js

```diff
@@ -31,14 +32,16 @@ const Filters = ({ namespace, baseParams, t }) => {
     }
   }, [dispatch, isReady]);
 
+  const selLen = selected.reduce((acc, x) => acc + x.values.length, 0);
   useEffect(() => {
-    if (isHydrated && isReady) {
+    if (isHydrated && isReady && prevSelRef.current !== selLen) {
```

Comment:
What is the fix here that did not work earlier?

Reply:
from one side I need fetch data when filters was change (add/remove checkbox) from other side I don't need if nothing was select (thats was happened before). For don't push to trigger dependency complicate object I use length of filters values for compare

Reply:
What is someone adds one checkbox and removes another, that way the length will be the same and you will not update anything?

Reply:
it will 2 different updates